spec/models/notification_spec.rb

```diff
@@ -106,13 +106,23 @@
         :level      => notification.notification_type.level,
         :created_at => notification.created_at,
         :text       => notification.notification_type.message,
-        :bindings   => a_hash_including(:initiator => a_hash_including(:text => user.name,
-                                                                       :link => a_hash_including(:id, :model)),
+        :bindings   => a_hash_including(:initiator => a_hash_including(:text => user.name),
                                         :extra     => a_hash_including(:text => 'information')
                                        )
       )
     end
 
+    context 'link_to is set' do
+      let(:notification) do
+        FactoryGirl.create(:notification, :initiator         => user,
+                                          :notification_type => FactoryGirl.create(:notification_type, :link_to => 'initiator'))
+      end
+
+      it 'contains the link to the initiator' do
+        expect(notification.to_h).to include(:bindings => a_hash_including(:link => a_hash_including(:model, :id)))
```

Comment:
Couldn't this match the user more specifically? I figure it makes the expectation that the user object originally passed in the `initiator` field should match the one in the link.

Maybe something like this:
`expect(notification.to_h).to include(:bindings => a_hash_including(:link => {:model => "user", :id => user.id}))`